functions.php

```diff
@@ -16,6 +16,8 @@ function create_posttype()
 {
     add_theme_support('post-thumbnails');
 
+    add_rewrite_rule('^([.+])/?', '^radioblog/$matches[1]', 'bottom');
```

Comment:
@josephpalma, can you explain this line of code?

Reply:
routing posts to /radioblog? what was it before/what is the purpose now?